src/Build.UnitTests/Graph/ProjectGraph_Tests.cs

```diff
@@ -13,6 +14,14 @@ namespace Microsoft.Build.Graph.UnitTests
 {
     public class ProjectGraphTests
     {
+        private const string ProjA = "A.proj";
```

Comment:
Hmm, these are hardcoded in the project content, but used as consts here.

Here's what I was planning for the next change I'm working on: https://github.com/dfederm/msbuild/blob/dfederm/graph-target-lists/src/Build.UnitTests/Graph/ProjectGraph_Tests.cs#L96

Basically just work with project numbers and have the helper methods add the "proj" suffixes. Also, I used numbers for projects and letters for targets.

I'm fine with changing to your style though if you find a way to reduce the duplication.

Or we can just leave this and do a UT refactor after the fact :)